Automattic/wp-calypso
# remove prettier pragma from sass files

#### Changes proposed in this Pull Request

* Remove prettier pragma `@format` from any sass file in /assets and /client

#### Testing instructions

* code: make sure the only code changes are removing the prettier pragma

stacked on https://github.com/Automattic/wp-calypso/pull/29403
